packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/platform_interface/google_maps_flutter_platform.dart

```diff
@@ -357,6 +371,11 @@ abstract class GoogleMapsFlutterPlatform extends PlatformInterface {
     throw UnimplementedError('onLongPress() has not been implemented.');
   }
 
+  /// A marker icon managed by [ClusterManager] has been tapped.
+  Stream<ClusterTapEvent> onClusterTap({required int mapId}) {
+    throw UnimplementedError('onClusterTap() has not been implemented.');
+  }
```

Comment:
Same.

(We occasionally have had issues where just having a test calling new methods highlighted a type export problem, for instance, so this isn't as pointless as it may seem 🙂 )